addon/components/-private/base-pop-menu.js

```diff
@@ -67,6 +68,14 @@ export default class BasePopMenuComponent extends Component {
   @type('string')
   rootElementSelector = '.ember-application';
 
+  /**
+   * User-provided modifiers to the popper.
+   * Documentation for popper modifiers can be found at https://popper.js.org/popper-documentation.html#modifiers
+   */
+  @argument
+  @type('object')
+  popperModifiers = {};
```

Comment:
We should default this to null since we'll be creating an object anyways in `_popperModifiers`, that way we don't create an extra object if unless it's passed in